### ISSUE TITLE
Boosting in the wrong direction, cannot escape orbit after that

### ISSUE THREAD

**Issue by Oyzuu** [kirub/gmtk-19#32]
When boosting from orbit, ship boosts on tangent and then it is impossible to exit orbit (boosted two times to get out).

Build: FPP_015

**Comment by kirub** [comment]
Duplicate #8

**Comment by Oyzuu** [comment]
Not a duplicate, wording could be more precise.

This is a bug, a game breaking one where the ship will boost in the wrong direction and then be stuck in orbit.

![IMG_20190813_111339477](https://user-images.githubusercontent.com/19710540/62930547-ed0c0500-bdbc-11e9-9e56-3a706c458b0f.jpg)

**Comment by kirub** [comment]
Yeah I know, it is a duplicate = same bug that leads to these 2 different behaviors ^^
